### PR TITLE
More complete migration guide

### PR DESCRIPTION
**Pull request by lasote**

### CODE REVIEW

File: migrating_to_2.0/general.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Conan 1.x uses one profile by default, to start using two profiles, please do the following:
+
+- Pass ``-pr:b=default`` in the command line to most commands.
+- Or set the variable ``core:default_build_profile=default`` at the :ref:`global.conf<global_conf>` file to apply it
+  always, automatically.
+
+Do not use ``os_build``, ``arch_build`` anywhere in your recipes or code.
+
+
+- Revisions
```
Comment (czoido) [inline migrating_to_2.0/general.rst:22]:
```suggestion
Revisions
---------
```

**Review by czoido — APPROVED** [review]
I think this is much better

**Review by franramirez688 — APPROVED** [review]
Great job @lasote ! It's looking quite clear and useful the migration guide 👏

**Review by memsharded — APPROVED** [review]
Fantastic

**Pull request merged**